frontend/src/components/NewShoot/NewShootDetails.vue

```diff
@@ -137,22 +137,34 @@ export default {
     ]),
     ...mapGetters([
       'sortedKubernetesVersions',
+      'defaultKubernetesVersionForCloudProfileName',
       'shootByNamespaceAndName',
       'projectList'
     ]),
     sortedKubernetesVersionsList () {
-      return this.sortedKubernetesVersions(this.cloudProfileName)
+      return filter(this.sortedKubernetesVersions(this.cloudProfileName), ({ isExpired }) => {
+        return !isExpired
+      })
     },
     versionHint () {
+      const version = find(this.sortedKubernetesVersionsList, { version: this.kubernetesVersion })
+      if (!version) {
+        return undefined
+      }
+      const hintText = []
+      if (version.expirationDate) {
+        hintText.push(`Kubernetes version expires on: ${version.expirationDateString}. Kubernetes update will be enforced after that date.`)
+      }
       if (this.updateK8sMaintenance && this.versionIsNotLatestPatch) {
-        return 'If you select a version which is not the latest patch version, you may want to disable automatic Kubernetes updates'
+        hintText.push('If you select a version which is not the latest patch version (except for preview versions), you should disable automatic Kubernetes updates')
+      }
+      if (version.isPreview) {
+        hintText.push('Preview versions have not yet undergone thorough testing. There is a higher probability of undiscovered issues and are therefore not yet recommended for production usage')
```

Comment:
```suggestion
        hintText.push('Preview versions have not yet undergone thorough testing. There is a higher probability of undiscovered issues and are therefore not recommended for production usage')
```

Reply:
I copied this from https://github.com/gardener/gardener/blob/05e7905e2cb40e2f9f70c466281614a2a31b0008/docs/operations/versioning.md
But we can leave `yet`out if you want.

Reply:
yes I would leave it out